pusher/src/Controller/BaseHttpController.ts

```diff
@@ -0,0 +1,45 @@
+import { Server } from "hyper-express";
+import Response from "hyper-express/types/components/http/Response";
+import axios from "axios";
+
+export class BaseHttpController {
+    constructor(protected app: Server) {
+        this.routes();
+    }
+
+    protected routes() {
```

Comment:
Maybe you should make this method "abstract" to force people extending this class to implement it?

Reply:
The class must be to an abstract to do this